src/Utilities/Microsoft.Build.Utilities.csproj

```diff
@@ -35,7 +35,11 @@
     <PackageReference Include="System.Security.Permissions" />
     <PackageReference Include="System.Text.Encoding.CodePages" />
   </ItemGroup>
-  
+
+  <ItemGroup Condition="$(TargetFrameworkIdentifier) != '.NETFramework'">
+    <Compile Remove="AppDomainIsolatedTask.cs" />
```

Comment:
Why change this? We have had it compiled in with `[Obsolete]` to help guide people away from using it, so this is a breaking change.